Sources/SwiftParser/Diagnostics/MissingNodesError.swift

```diff
@@ -137,17 +140,42 @@ fileprivate extension TokenKind {
   }
 }
 
+/// Checks whether a node contains any tokens (missing or present)
+fileprivate class HasTokenChecker: SyntaxAnyVisitor {
```

Comment:
We really need to define a better visitor interface for all this. Ie. one that returns a result + has stop.